packages/rootage/components/radiomark.yaml

```
@@ -0,0 +1,56 @@
+      disabled:
+        icon:
+          size: $dimension.x3
+    size=medium:
+      enabled:
+        root:
+          size: $dimension.x5
+        icon:
+          size: $dimension.x2
+      disabled:
+        icon:
+          size: $dimension.x2_5 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)